#5 - Histórias do Cardápio

henrilhos opened the pull request:
## Descrição

Adicionada as histórias do cardápio na documentação
<!-- Explicar sobre seu PR e quais mudanças foram feitas -->

## Issues relacionadas

<!-- Adicionar o número da issue que será fechada referente as mudanças do PR -->

Resolve #5 

<!-- ## Informações adicionais -->

<!-- Inserir outras informações -->

henrilhos committed:
WIP: Historias do Cardapio

henrilhos commented:
alguém do @restaurante-popular-catolicasc/cinza favor terminar

henrilhos committed:
Correçao gramatical

henrilhos committed:
Merge branch 'master' into feature/historia-cardapio

LDinizz commented on historias-de-usuarios/cardapio.md:
```diff
@@ -0,0 +1,24 @@
+#### Quero
+
+- Montar fichas técnicas para que sejam seguidas pelas cozinheiras, essas receitas devem ser armazenadas para que possam ser impressas posteriormente e reutilizadas sempre que necessário;
+- Que as ordens de produção sejam impressas com nas quantidades de cada ficha técnica para que a cozinheira já possa retirar, junto com a ficha técnica, para dar baixa no estoque;
```
Pelo que eu entendo do negócio, quem dá baixa no estoque e pega os itens é o Estoquista. Ou seja, as cozinheiras já recebem o material necessário para preparar a receita.
@gregori Dê sua opinião no processo.

LDinizz commented on historias-de-usuarios/cardapio.md:
```diff
@@ -0,0 +1,24 @@
+
+#### Quero
+
+- Receber a ficha técnica e a receita do dia impressa, ir até o estoque já com a ordem de produção retirada com a nutricionista para que o estoquista me forneça os produtos que preciso para a receita.
```
Mesma situação acima. Do ponto de vista do processo, o estoquista já teria deixado todos os itens na cozinha não?

henrilhos commented:
@Clanger72 abri novamente o teu PR, com algumas coisas ajustadas

Clanger72 committed:
Alteração do tópico cozinheira 

Alteração do tópico cozinheira após entendimento de que o processo de retirada e entrega dos produtos necessários para aquele dia são feitos pelo estoquista e não pela cozinheira.
